.github/workflows/go_test.yml

```diff
@@ -37,3 +37,22 @@ jobs:
           mysql -e "CREATE DATABASE IF NOT EXISTS $DB_DATABASE;" -u$DB_USER -p$DB_PASSWORD
       - name: Test with Go
         run: go test -v -race ./storage/mysql/... -is_mysql_test_optional=false
+
+  test-aws-mysql:
+    env:
+      DB_DATABASE: test_tessera
+      DB_USER: root
+      DB_PASSWORD: root
+
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Start MySQL
+        run: |
+          sudo /etc/init.d/mysql start
+          mysql -e "CREATE DATABASE IF NOT EXISTS $DB_DATABASE;" -u$DB_USER -p$DB_PASSWORD
```

Comment:
This is not the best way to start the MySQL service. I will update the one above soon.

https://docs.github.com/en/actions/use-cases-and-examples/using-containerized-services/creating-postgresql-service-containers

Reply:
Ah, my bad. Please ignore my comment. That's for PostgreSQL. The MySQL is already pre-installed in the ubuntu-latest image.